modules/partials/logging-dashboard-widget.php

```diff
@@ -5,14 +5,43 @@
  * @since 1.0.0
  */
 
+wp_enqueue_style( 'ilr-login-style', ILR_URL . '/lib/css/ilr-admin.css' );
```

Comment:
@brothman01 If you want to grant me write access to your repo I can make changes to the pull request before I merge it.